docsource/modules150-160.rst

```diff
@@ -22,7 +22,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |del| account_edi_ubl_bis3                      |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| account_edi_ubl_cii                             |                      |                                                 |
+| account_edi_ubl_cii                             |Done                  |Merged into account_edi_ubl_cii                  |
```

Comment:
@NgoHoangHung I see that you are confusing something here. Please check again

Reply:
sorry, this is huge my mistake.